solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -115,11 +116,46 @@ pub mod solana_ibc {
         ctx.accounts.chain.set_stake((*ctx.accounts.sender.key).into(), amount)
     }
 
+    /// Called to set up escrow and mint accounts for given channel and denom.
+    #[allow(unused_variables)]
+    pub fn deliver_init_escrow<'a, 'info>(
+        ctx: Context<'a, 'a, 'a, 'info, DeliverInitEscrow<'info>>,
+        port_id: ibc::PortId,
+        channel_id_on_b: ibc::ChannelId,
+        base_denom: String,
```

Comment:
Perhaps just `init_escrow`?

```suggestion
    pub fn init_escrow<'a, 'info>(
        _ctx: Context<'a, 'a, 'a, 'info, DeliverInitEscrow<'info>>,
        _port_id: ibc::PortId,
        _channel_id_on_b: ibc::ChannelId,
        _base_denom: String,
```